### PR TITLE
RDoc-2049-Updated image and created example to demonstrate how to set

### PR DESCRIPTION
**Pull request by serezhub**

### CODE REVIEW

**Comment by Danielle9897** [comment]
@serezhub
I see the studio issue this was fixed in is **5.2**
Why the files in this PR are for **5.0** ?

**Review by Danielle9897 — CHANGES_REQUESTED** [review]

File: Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown [hunk 1]
```diff
@@ -72,6 +69,54 @@
...
+Rollup and retention policies help you manage the amount of data in storage by 
+setting how close you want to zoom in and out in different time frames.  
+[See Heart Clinic Example.](../../../studio/database/settings/time-series-settings#heart-clinic-example)
+
+**Rollup Policies**  
+Rollup aggregates the current data to summarize it and zoom out 
+to a bigger picture with a lower resolution that uses less storage space.  
+
+You can set different levels of resolution by adding new policies that further aggregate 
+the previously aggregated data.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown:84]: done

File: Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown [hunk 2]
```diff
@@ -72,6 +69,54 @@
...
+Rollup aggregates the current data to summarize it and zoom out 
+to a bigger picture with a lower resolution that uses less storage space.  
+
+You can set different levels of resolution by adding new policies that further aggregate 
+the previously aggregated data.  
+
+**Retention Policies**  
+Set the amount of time that time series data is kept before being deleted.  
+You can set a different retention policy for each rollup 
+so that you can delete the previous data after it has been summarized.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown:89]: done

File: Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown [hunk 3]
```diff
@@ -20,12 +20,9 @@
       A retention period defines how long time series entries are kept.  
       An entry that exceeds this period will be removed.  
 
-
-
-
-
 * In this page:  
   * [Time Series Settings View](../../../studio/database/settings/time-series-settings#time-series-settings-view)  
+  * [Rationale for Configuring Time Series](../../../studio/database/settings/time-series-settings#rationale-for-configuring-time-series)
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown:25]: done

File: Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown [hunk 4]
```diff
@@ -72,6 +69,54 @@
 
 {PANEL/}
 
+{PANEL: Rationale for Configuring Time Series}
+
+Many applications continuously store massive amounts of time series data.  
+Rollup and retention policies help you manage the amount of data in storage by 
+setting how close you want to zoom in and out in different time frames.  
+[See Heart Clinic Example.](../../../studio/database/settings/time-series-settings#heart-clinic-example)
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown:77]: done

> ... It must be kept at least until it is aggregated/summarized in the rollup policy, 
but can be set to be available for a period after the rollup, as seen in the example below.

File: Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown [hunk 5]
```diff
@@ -72,6 +69,54 @@
...
+
+**Retention Policies**  
+Set the amount of time that time series data is kept before being deleted.  
+You can set a different retention policy for each rollup 
+so that you can delete the previous data after it has been summarized.
+
+{NOTE: }
+#### Heart Clinic Example  
+Imagine a heart clinic whose raw data shows their patients' heart rates every minute. 
+Here are the different levels of resolution that they want to see:
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown:94]: done

File: Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown [hunk 6]
```diff
@@ -13,23 +13,69 @@
...
+
+Many applications continuously store massive amounts of time series data.  
+
+Rollup policies help you manage the amount of data in storage by 
+setting how close you want to zoom in and out in different time frames.  
+
+Retention time periods then allow you to decide for how long to keep the higher resolution data in storage before it is deleted, 
+or if to delete it at all.  
+It must be kept at least until it is aggregated/summarized in the rollup policy, 
+but can be set to be available for a period after the rollup, as seen in the example below.
```
Comment (Danielle9897) [inline Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown:48]: for a period
=> 
for some time period

Comment (serezhub) [reply Documentation/5.0/Raven.Documentation.Pages/studio/database/settings/time-series-settings.markdown:48]: done 

> but can be set to be available for any time period after the rollup

**Comment by Danielle9897** [comment]
* This PR is approved in the new PR:
https://github.com/ravendb/docs/pull/1553

* @serezhub you can close this PR

**Pull request closed without merging**